http/1.5/modules/ROOT/pages/http-connector-examples.adoc

```diff
@@ -0,0 +1,35 @@
+= HTTP Connector Examples - Mule 4
+:keywords: anypoint, connectors, transports
+
+To get full advantage of Anypoint Connector for HTTP (HTTP Connector), check the following examples that help you to configure the connector both in Anypoint Studio and Design Center.
+
+
+== HTTP Connector Studio Examples
```

Comment:
If all the examples are in Studio, probably don't need to mention Studio in the title of this section.

Reply:
Is there a corresponding topic like this only the examples are in Design Center?

Reply:
Yes, they are at the following section of this page called:
== HTTP Connector Design Center Examples

Reply:
Oh, haha I totally missed that! :-)